book/A-git-in-other-environments/sections/powershell.asc

```diff
@@ -49,26 +56,31 @@ Then you can go back and try again.
 This happens, because the modules that ship with Windows PowerShell are signed with a different publishment certificate.
 
 ===== Update PowerShell Prompt
+
 To include git information in your prompt, the posh-git module needs to be imported.
-To have posh-git imported every time PowerShell starts, execute the Add-PoshGitToProfile command which will add the import statement into you $profile script.
+To have posh-git imported every time PowerShell starts, execute the `Add-PoshGitToProfile` command which will add the import statement into you `$profile` script.
```

Comment:
Here's another typo I noticed while reviewing this. Mind including this in your fix?

```suggestion
To have posh-git imported every time PowerShell starts, execute the `Add-PoshGitToProfile` command which will add the import statement into your `$profile` script.
```

Reply:
🤦